3rdPartyCypress/cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+	"video": false,
+	"screenshotOnRunFailure": false
```

Comment:
Indentation is not consistent with current `cypress.json`.

Reply:
Good spot, pushing fix now.